README.rst

```diff
@@ -1,4 +1,9 @@
-Django Vaultkeeper Daemon Repository
-========================
+vaultkeeper
+============
 
+Installing the Package
+~~~~~~~~~~~~~~~~~~~~~~
 
+Install the package for development with the following command:
+
+`pip install -e . [testing] --process-dependency-links`
```

Comment:
This needs to be `-e .[testing]` (no space). (In some shells you might need to quote it so the `[]` doesn't get interpreted as a glob pattern.)